tests/lib/rules/indent.js

```diff
@@ -185,7 +202,7 @@ ruleTester.run("indent", rule, {
         {
             code:
             "require('http').request({hostname: 'localhost',\n" +
-            "                         port: 80}, function(res) {\n" +
+            "  port: 80}, function(res) {\n" +
```

Comment:
IMO and confirmed by @ljharb and @not-an-aardvark this is the correct indenting functionality.